swagger/swagger.yaml

```diff
@@ -28,11 +28,113 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+
+  /:      
+    post:
+      summary: Lists all listings
+      operationId: listListings
+      tags:
+      - listings
+      responses:
+        '201':
+          description: Array of listings
+          content:
+            application/json:    
+              schema:
+                $ref: "#/components/schemas/ResponseListings"
+        '401':
+          description: Array of listings
+          content:
+            application/json:    
+              schema:
+                $ref: "#/components/schemas/ResponseListings"
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+
+  /:id:      
+    delete:
+      summary: Lists all listings
+      operationId: listListings
+      tags:
+      - listings
+      responses:
+        '200':
+          description: Array of listings
+          content:
+            application/json:    
+              schema:
+                $ref: "#/components/schemas/ResponseListings"
```

Comment:
ResponseListings -> ResponseListing
är ett objekt du får tillbaka, inte en array

uppdatera beskrivningen med, det är itne en array of listings